storage/pebble/operation/batch.go

```diff
@@ -0,0 +1,77 @@
+package operation
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/cockroachdb/pebble"
+
+	"github.com/onflow/flow-go/storage"
+)
+
+// batchWriter wraps the storage.BatchWriter to make it compatible with pebble.Writer
+type batchWriter struct {
+	batch storage.BatchWriter
+}
+
+func NewBatchWriter(batch storage.BatchWriter) pebble.Writer {
+	return batchWriter{batch: batch}
+}
+
+// pebble.Writer implementation
+func (b batchWriter) Set(key, value []byte, o *pebble.WriteOptions) error {
+	return b.batch.Set(key, value)
+}
+
+func (b batchWriter) Delete(key []byte, o *pebble.WriteOptions) error {
+	return b.batch.Delete(key)
+}
```

Comment:
Please provide minimal documentation. Highlighting that `Set` silently overrides existing values and `Delete` silently ignores non-existing keys is worth-while emphasizing in my opinion.